### DISCUSSION TITLE
Qobuz plugin failed to decode

### DISCUSSION THREAD

**Discussion by ghost** [MusicPlayerDaemon/MPD#1220] [Q&A]
I'm running archlinux with ffmpeg installed but qobuz plugin still don't work for me.

I add a track with `mpc add qobuz://track/23601296` 
In ncmpc it shows like ` 00 (artist n/a) - (title n/a) - (N/A) ` and i can't play it, when it try:

```
decoder_thread: probing plugin ffmpeg
exception: Failed to decode qobuz://track/23601296; avformat_open_input() failed: Invalid data found when processing input
player: played "qobuz://track/23601296"
playlist: stop
```

My mpd.conf
```
input {
        enabled "yes"
        plugin "qobuz"
                app_id "XXXXXXXXX"
                app_secret ",32 char hash,another"
                username "myemail addres"
                password "my plaintext password"
                format_id "5"
}
```

I analyze network traffic and only got dns resolve successfully to qobuz nothing more. Do i need more open ports except 443?

**Comment by MaxKellermann** [comment]
I guess the Qobuz plugin is broken after Qobuz changed the protocol, see https://github.com/MusicPlayerDaemon/MPD/issues/1209
